src/handbook/product/pricing.md

```diff
@@ -40,17 +50,10 @@ The team is billed per device at the same rate regardless of tier or number of d
 
 Devices are part of the licensed instances. The limit in the open source edition is 50 devices.
 
-## Add-Ons
-
-There's currently no plan for add-ons. Add-ons might be used in the future to
-allow overage on resource consumption, for example when storing files.
-Where possible add-ons should be avoided in favor of additional features.
-
 ## Licenses
 
-From the perspective of licensing the open source and FlowForge Premium are
-different tiers of the same product. The open source edition doesn't require a
-license key to be uploaded. Without a valid license a basic set of features and
+From the perspective of licensing the open source and FlowForge Premium, and FlowForge Enterprise are different tiers of the same product.
```

Comment:
```suggestion
From the perspective of licensing the there's an unlicensed edition: open-source. This version is always self-managed. There's also a licensed version of FlowForge, this can be self-managed or FlowForge Managed. For FlowForge managed properties there's 3 tiers; Starter, Premium, and Enterprise.
```